build.xml

```diff
@@ -12,9 +13,21 @@
 		phpstan
 	"/>
 
+	<target name="composer-diagnose">
```

Comment:
@ondrejmirtes 

Not sure, should I add a task to download `composer.phar` instead?

I think it would be useful for being able to run `phing` locally, too. 

What do you think?